applications/conversations/views/messages/popin.php

```diff
@@ -45,7 +45,7 @@
         <?php endforeach; ?>
         <li class="Item Center">
             <?php
-            echo anchor(sprintf(t('All %s'), t('Messages')), '/messages/inbox');
+            echo anchor(t('All Messages'), '/messages/inbox');
```

Comment:
Is there already a translation for "All Messages"? If not, this is going to break this translation everywhere.

Reply:
Make the change backwards compatible instead by using the existing version as the fallback.